README.rst

```diff
@@ -154,6 +154,9 @@ use-cases. Simply install them to make them available:
 - `keyrings.alt <https://pypi.org/project/keyrings.alt>`_ - "alternate",
   less common backends, originally part of the core package, but now
   available for opt-in.
+- `keyring_jeepney <https://pypi.python.org/pypi/keyring_jeepney>`__ - a
```

Comment:
Yep, I'm in the habit of using two underscores to avoid Sphinx giving me warnings if there are two links with the same text. That's not going to be a problem here, though.